src/Serialization.purs

```diff
@@ -891,3 +888,7 @@ hashScriptData cms rs ps = do
     [] -> _hashScriptDataNoDatums rs' cms'
     _ -> _hashScriptData rs' cms' =<< fromJustEff "failed to convert datums"
       (traverse convertPlutusData ps)
+
+serialiseData :: forall (a :: Type). ToData a => a -> Maybe CborBytes
```

Comment:
We need to re-export this function as part of `Contract` interface. I think `Contract.PlutusData` is a good place for it.

`deserializeData` should go there as well.